src/Autocomplete.js

```diff
@@ -9,14 +10,21 @@ class Autocomplete extends Component {
     super(props);
 
     this.state = {
-      value: ''
+      value: props.value || ''
     };
 
     this.renderIcon = this.renderIcon.bind(this);
     this.renderDropdown = this.renderDropdown.bind(this);
     this._onChange = this._onChange.bind(this);
   }
 
+  componentWillReceiveProps (nextProps) {
```

Comment:
You could even do `componentWillReceiveProps ({ value }) {`

Reply:
p.s. doesn't the linter complain about space between parens and function name?

Reply:
Yep, I'll do it and I'll correct the parens

Reply:
Actually, it seems the parens are like this everywhere? 
The linter does complain when it's missing :)

Reply:
Absolutely correct, I was confused with another project where I have the opposite settings :D